packages/graph/hash_graph/lib/graph/src/knowledge/link/mod.rs

```diff
@@ -0,0 +1,165 @@
+use std::{collections::HashMap, fmt};
+
+use postgres_types::ToSql;
+use serde::{Deserialize, Serialize};
+use utoipa::Component;
+
+use super::EntityId;
+use crate::ontology::types::uri::VersionedUri;
+
+/// A Link between a source and a target entity identified by [`EntityId`]s.
+/// The link is described by a link type [`VersionedUri`].
```

Comment:
rustdoc will use the first paragraph as a summary on the index page of the module, this will render it more nicely.
```suggestion
/// A Link between a source and a target entity identified by [`EntityId`]s.
///
/// The link is described by a link type [`VersionedUri`].
```